src/OpenTK.Windowing.Common/Input/Image.cs

```diff
@@ -0,0 +1,63 @@
+//
+// WindowIcon.cs
+//
+// Copyright (C) 2018 OpenTK
+//
+// This software may be modified and distributed under the terms
+// of the MIT license. See the LICENSE file for details.
+//
+
+using System;
+
+namespace OpenToolkit.Windowing.Common.Input
```

Comment:
Why is this in Input?

Reply:
It was made for WindowIcon, which was in the same namespace.

Honestly, I feel like this would be better off as part of OpenTK.Core? Since it could, theoretically, have uses beyond just windowing/mouse cursors.

Reply:
Alright, put it in a drawing subfolder in Core.

Reply:
Should it be in a sub-namespace too?

I'm not sure about putting it in Core.Drawing, since we'd be making a folder/namespace entirely for a single class - I can't see us making anything else that would go there in the forseeable future.

Reply:
It's best to open a new namespace only when the types in that namespace belong to a separate domain.